core/ontology/generis.rdf

```diff
@@ -40,6 +40,14 @@
     <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#Boolean"/>
     <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#RadioBox"/>
   </rdf:Description>
+  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/generis.rdf#is_statistical">
+      <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
```

Comment:
I think, we should edit the same indentation, as other lines have: 2 spaces 🙂

Reply:
Done, thanks! :)